src/Button/Button.scss

```diff
@@ -204,6 +204,18 @@
   }
 }
 
+.btn,
+.btn-primary,
+.btn-group-lg>.btn-primary.btn,
+.btn-group-sm>.btn-primary.btn,
+body .page-link {
```

Comment:
[clarification] Do you mind shining some light on how you arrived at these selectors? For example, why does `.page-link` use the `button-size` mixin and whether each button variant will need to be included beyond just `btn-primary`?

Reply:
I did some research on the whole project and found out that using only one `btn-primary` selector does not produce the required focus state. All these selectors were found during the check of all components and their use is mandatory. I also decided to use the `button-size` mixin which we apply to the `before` pseudo-element and have the ability to get a dynamic change in the border-radius parameters when the Paragon theme changes.

Reply:
@PKulkoRaccoonGang I'm still not entirely following the approach here. It's not just the one `.btn-primary` selector without the focus state. All the variants have a rounded border radius:

<img width="701" alt="image" src="https://user-images.githubusercontent.com/2828721/163564221-c195c216-511a-4043-b285-a8ef704076bf.png">

Because `.btn` is being used as a selector, it applies to all buttons which is likely why your solution technically works, even though it doesn't need the specificity you are providing with regard to a specific variant.

Could the following be added in the already existing selector for `.btn` above?

```scss
.btn {
  @include button-size($btn-padding-y, $btn-padding-x, $btn-font-size, $btn-line-height, $btn-border-radius);
  
  // ...

  &:focus:before {
    @include button-size($btn-padding-y, $btn-padding-x, $btn-font-size, $btn-line-height, $btn-border-radius);
  }
}
```

This solves the issue locally for me.